dbus-codegen/src/main.rs

```diff
@@ -56,6 +58,10 @@ Defaults to 'RefClosure'."))
         let c = dbus::Connection::get_private(bus).unwrap();
         let p = c.with_path(dest, path, 10000);
         p.introspect().unwrap()
+    } else if let Some(_file_path) = matches.value_of("file")  {
```

Comment:
Why ignore the path with `_` prefix if you will use it in the block?

Reply:
I think we should error out if both `destination` and `file` arguments are provided as otherwise it's ambiguous, which will be used and we really don't need to support the case of both being provided. @diwic wouldn't you agree?

Reply:
I guess I prefer erroring out, but it's not a big deal.

Reply:
@zeenix @diwic The existing code looks for input whenever `destination` is not provided. Do I have too still keep that code such that it executes if both `destination` and `file` are not provided?

Reply:
@inzamam-iqbal The code should behave like this:

 * Both destination and file - panic with relevant error message
 * destination only - read from destination (like today)
 * file only - read from file
 * neither file, nor destination - read from stdin (like today)

Does that answer your question?

Reply:
@diwic yes I have updated the PR like that